lib/spack/env/cc

```diff
@@ -191,11 +202,11 @@ while [ -n "$1" ]; do
         -Xlinker,*)
```

Comment:
I'm not familiar with `-Xlinker`'s syntax details, and I'm not using it myself. I'm hesitant to change something that I can't check for correctness.